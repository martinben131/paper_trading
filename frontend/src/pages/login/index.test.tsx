import React from "react";
import { shallow } from "enzyme";
import Login from "./index";

test("Renders login page correctly", () => {
  const component = shallow(<Login />);
  expect(component).toMatchSnapshot();
});
