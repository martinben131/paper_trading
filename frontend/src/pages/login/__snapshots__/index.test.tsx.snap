// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders login page correctly 1`] = `
<Fragment>
  <h4
    className="bp3-heading"
  >
    Login
  </h4>
  <Blueprint3.FormGroup>
    <Blueprint3.InputGroup
      id="username"
      onChange={[Function]}
      placeholder="Enter email here"
    />
    <br />
    <Blueprint3.InputGroup
      id="password"
      onChange={[Function]}
      placeholder="Enter password here"
      type="password"
    />
    <br />
    <Blueprint3.Button
      disabled={true}
      intent="success"
      onClick={[Function]}
      text="Submit"
    />
  </Blueprint3.FormGroup>
</Fragment>
`;
